gdb-ctest

```diff
@@ -77,6 +77,11 @@ get_test_property()
     get_test_json "$1" | jq -r ".properties | map(select(.name == \"$2\"))[].value | join(\" \")"
 }
 
+get_test_workdir()
+{
+    get_test_json "$1" | jq -r ".properties | map(select(.name == \"WORKING_DIRECTORYz\"))[].value"
```

Comment:
Is this a typo? `WORKING_DIRECTORYz`

Also, I'm not really sure, why there is a `join` in `get_test_property`, but if I got rid of that, you could implement `get_test_workdir` in terms of that. I don't really care that much though.

Reply:
In the other PR you removed `get_test_environment` so I guess you could do without `get_test_workdir`...

Reply:
> Is this a typo? `WORKING_DIRECTORYz`

Sorry, pushed a wrong file with a testing leftover. Fixed.
 
> Also, I'm not really sure, why there is a `join` in `get_test_property`, but if I got rid of that, you could implement `get_test_workdir` in terms of that. I don't really care that much though.

I haven't tested it (this project has no fixtures), but I guess you need that join for multi-value bits (CMake arrays). I just know that the script used to work with these features when I last tried it, and that simply calling `get_test_property` on a string value fails.

> In the other PR you removed `get_test_environment` so I guess you could do without `get_test_workdir`...

OK, so I changed this into a `get_test_property_str`.